.gitignore

```diff
@@ -17,3 +17,4 @@ sphinx/build/
 docs/autosummary/
 .DS_Store
 
+.cache
```

Comment:
Any reason for this one? Seems unrelated to the rest.

Reply:
Sneaked in there from a `git status` annoyance. Can nuke or leave...

Reply:
Think I'd rather leave it out of this PR for clarity (can you amend commit and force-push so it's still one commit?) but happy to add it in another PR if it seems generally useful, e.g. if it's something that's likely to be created on most people's systems and not just something that's happening to you.

Reply:
I think I did that right, please check.

Reply:
Yep, looks good here.